pkg/cmd/codespace/create.go

```diff
@@ -113,12 +114,17 @@ func (a *App) Create(ctx context.Context, opts createOptions) error {
 
 	promptForRepoAndBranch := userInputs.Repository == ""
 	if promptForRepoAndBranch {
+		currentRepo, err := a.baseRepo()
+		if err != nil {
+			return err
+		}
```

Comment:
I tried this out and had a couple pain points:

- If you're not in any repo, then the command just fails - I would have expected it to fall back on existing behavior

![image](https://user-images.githubusercontent.com/5447118/204625943-080e9088-5f79-4c31-8bbc-6e863ad792df.png)

- If you *are* in a repo, it seems like you can only choose that repo, or a related remote - I expected that it would use the default text, but allow you to clear it out and use any repo

![image](https://user-images.githubusercontent.com/5447118/204626812-fc48204e-8d45-4271-ad59-70cdc986a3a0.png)

Reply:
Thanks for taking a look, I completely agree with you about both of these issues. I'll take a look at improving the PR in line with your suggestions.

Reply:
> If you're not in any repo, then the command just fails - I would have expected it to fall back on existing behavior

I've fixed this :+1: 

> If you are in a repo, it seems like you can only choose that repo, or a related remote - I expected that it would use the default text, but allow you to clear it out and use any repo

I've experimented and this is actually a generic prompt used by all commands (e.g. try running `gh pr list`) the first time that you run any command in a checkout with multiple remotes. If you choose one then you can proceed.